### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by Flouse**
## New issue templates
- bug report
- feature request

---

- [x] It is difficult for reporters to get the prebuild-images versions. We should teach them how to get this information. For example, `kicker info` or use `gw_get_node_info` RPC.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: bug
+assignees: ''
+
+---
+
```
Comment (Flouse) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]:
```suggestion

**Version**
- Godwoken v1 or v0?
- The prebuild-image you are using or the commit-sha1

```

**Review by classicalliu — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+**Version**
+- Godwoken v1 or v0?
+- The prebuild-image you are using or the commit-sha1
```
Comment (keroro520) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]: It is difficult for reporters to get the prebuild-images versions. We should teach them how to get this information. For example, `kicker info` or use `gw_get_node_info` RPC.

**Review by keroro520 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (Flouse) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]:
```suggestion
- (Optional) Get the versions from `poly_version` RPC
<details>
<summary>command example</summary>

```sh
curl https://godwoken-testnet-v1.ckbapp.dev -X POST \
  -H "Content-Type: application/json" \
  -d '{"jsonrpc": "2.0", "method":"poly_version", "params": [], "id": 1}'
```
</details>

```

Comment (Flouse) [reply .github/ISSUE_TEMPLATE/bug_report.md]: https://github.com/nervosnetwork/godwoken-web3/pull/290/commits/d8660e2412682b35f52d6fb72efc1f5026877482

**Review by keroro520 — APPROVED** [review]

**Pull request merged**